### ISSUE TITLE
_ in URL interpreted as italic format

### ISSUE THREAD

**Issue by paulgirard** [medialab/fonio#333]
**Describe the bug**
underscores in URL are interpreted as italic 

**To Reproduce**
1. enter this URL in any text section: http://ricardo.medialab.sciences-po.fr/WTW/Lund_seminar_20181010/#/20
2. type anything (but newline) right after it
3. seminar is now in italic, underscors disapeared

![image](https://user-images.githubusercontent.com/193478/82812942-611f2180-9e94-11ea-86f5-f4f6db9c86ed.png)
became
![image](https://user-images.githubusercontent.com/193478/82812976-72682e00-9e94-11ea-88d2-5579bd470444.png)

**Expected behavior**

Don't auto-format URL

Well, know that I understood that I can create a "web page resource" in the library I don't know if that's really a bug ?

**Comment by robindemourat** [comment]
Yes that's not a bug that's a feature ;) - due to markdown shortcuts. Even if the preferred way is to create a webpage entity, this issue could be addressed by:

* refactoring markdown shortcuts to avoid being triggered while being in middle of a word
* implementing a URL detection mechanism (during pasting ? during any text input ?) to automatically create web page resources out of pasted plain text URLs
